x-pack/plugins/uptime/public/components/overview/monitor_list/columns/translations.ts

```diff
@@ -8,11 +8,11 @@
 import { i18n } from '@kbn/i18n';
 
 export const ENABLE_STATUS_ALERT = i18n.translate('xpack.uptime.monitorList.enableDownAlert', {
-  defaultMessage: 'Enable status alert',
+  defaultMessage: 'Enable rule',
```

Comment:
Do we lose some context here? Can we call it `Enable status rule` or `Enable status check rule`?

Reply:
I understand what you mean, I had the same thought but was wondering if it is ok as the column header says `Status alert`.